sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/KeyVaultBackupAsyncClient.java

```diff
@@ -554,6 +652,117 @@ static Mono<PollResponse<KeyVaultRestoreOperation>> processRestoreOperationRespo
             toLongRunningOperationStatus(operationStatus.toLowerCase(Locale.US)), response.getValue()));
     }
 
+    /**
+     * Initiates a pre-full restore of the Key Vault. This operation checks if it is possible to restore an entire
```

Comment:
Nit: I think "full pre-restore" or "pre-restore of the full Key Vault" sounds a bit nicer than "pre-full restore", but that could just be me. I noticed that Python's generated operation uses "pre_full_restore" in the method name, but uses "pre-restore for a full restore" in docstrings.

Reply:
You're right, it reads a bit odd. I'll change it to something nicer :)